tools/iree-dump-parameters-main.c

```diff
@@ -120,6 +120,13 @@ int main(int argc, char** argv) {
                        "Dumps information about parsed parameter files.\n");
   iree_flags_parse_checked(IREE_FLAGS_PARSE_MODE_DEFAULT, &argc, &argv);
 
+  if (argc > 1) {
+      fprintf(stderr, "Error: no positional arguments expected.\n");
```

Comment:
you'll need to run clang-format (the bots are running now and will give you a diff, but if you'll be making changes you'll want to set it up locally - you can use https://github.com/openxla/iree/blob/main/build_tools/scripts/lint.sh to do what the bots do, but configuring your editor to format on save is best)

Reply:
Sorry, missed this one, I usually run format separately from save, thanks for the tip :)